backend/src/resolvers/Mutation/Workspace.js

```diff
@@ -57,6 +57,32 @@ query($id : ID!) {
 }
 `
 
+const bloom = [{
+  name: 'REMEMBER',
+  type: 'bloom',
+  priority: 0
```

Comment:
Remember could have a higher priority to make it different from custom tags.

Reply:
Negative priorities are allowed here so it should be fine as is